### DISCUSSION TITLE
Merge local (img) and server users

### DISCUSSION THREAD

**Discussion by Soylor** [ltsp/ltsp#790] [Q&A] [answered]
I'll like to:

- **Allow only certain server users** for login (using sshfs and accesing server home)
- **Allow img local users** (using local home)

Can be achieved using **PWMERGE_**  ? (Wich one and what kind of regular expression sintax?)
Do i need to use **ltsp image MyImage --cleanup=0**  ??

Any help is wellcome?

**Comment by Soylor** [comment]
I tried CLEANUP=0 for trying to keep users inside the chroot to be inside the generated image but does not work.

**Comment by alkisg** [comment]
You need PWMERGE_, but I don't have time to tell you the exact regex, see the man page and previous discussions here.

**Reply by Soylor** [reply]
PWMERGE is for adding/preserve users from server or from image?

**Reply by Soylor** [reply]
Image users Called **administrator**, **alfa** and **localuser**
Server User called **user01,...user10**
For preserving **localuser** and server users **user01 to user10**? what do i need?
PWMERGE_???=

**Comment by Soylor** [comment, marked as answer]
Tried 

`ltsp image jammy cleanup=0`
and
```
PWMERGE_DUR="localuser"
```
Did the work, thanks.
